packages/table/table.scss

```diff
@@ -0,0 +1,70 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+$bottom-border-size: 1px;
+
+.jkl-table {
+    border-collapse: collapse;
+
+    &__anchor {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        opacity: 0;
+        top: 0;
+        left: 0;
+    }
+
+    &__heading {
+        color: $matt-svart;
+        font-size: $font-size-1;
+        font-weight: normal;
+        line-height: $line-height-1;
+        padding: $component-spacing--small 0.5rem;
+    }
+
+    &__data-cell {
+        position: relative;
+        box-sizing: border-box;
+        padding: 1.5rem 0.5rem;
+        line-height: $line-height-2;
+    }
+
+    &__heading,
+    &__data-cell {
+        text-align: left;
+        vertical-align: top;
+
+        &:first-of-type {
+            padding-left: 1rem;
+        }
+
+        &:last-of-type {
+            padding-right: 1rem;
+        }
+    }
+
+    &__row {
+        box-shadow: 0 ($bottom-border-size * 2) 0px -1px rgba(0, 0, 0, 0.2);
+
+        &--anchor-row {
+            cursor: pointer;
+        }
+
+        &--data-row {
+            transition: background-color 150ms ease, box-shadow 150ms ease;
+            &:hover {
```

Comment:
Should we distinguish between normal and clickable rows somehow? It feels a bit unnatural not to be able to click on regular rows with the current style, and the changing cursor is kind of a weak distinction IMO.

Reply:
Good catch. This was a misunderstanding between me and the designer. There should be no hover state on normal rows.